lib/wigets/help_dialog.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:flutter/services.dart' show rootBundle;
+
+import 'package:fokus/utils/app_locales.dart';
+
+class HelpDialog extends StatelessWidget {
+  final String helpPage;
+
+  HelpDialog({
+    @required this.helpPage
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Dialog(
+      shape: RoundedRectangleBorder(),
+      elevation: 0.0,
+      backgroundColor: Colors.transparent,
+      child: dialogContent(context)
+    );
+  }
+
+  Widget dialogContent(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.symmetric(horizontal: 8.0, vertical: 12.0),
+      decoration: new BoxDecoration(
+        color: Colors.white,
+        shape: BoxShape.rectangle,
+        borderRadius: BorderRadius.circular(4.0)
+      ),
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        children: <Widget>[
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              Padding(
+                padding: EdgeInsets.only(left: 8.0, right: 10.0, top: 2.0),
+                child: Icon(Icons.help_outline)
+              ),
+              Text('${AppLocales.of(context).translate("help." + helpPage)}', style: Theme.of(context).textTheme.headline2)
```

Comment:
translate() already returns a string - you don't need to interpolate it ;)

Reply:
Ups 👀